packages/dx-react-scheduler/src/plugins/editing-state.tsx

```diff
@@ -28,7 +28,7 @@ class EditingStateBase extends React.PureComponent<EditingStateProps, EditingSta
   commitDeletedAppointment: ActionFn<any>;
 
   static defaultProps: Partial<EditingStateProps> = {
-    defaultEditingAppointment: {},
+    defaultEditingAppointment: undefined,
     defaultAppointmentChanges: {},
     defaultAddedAppointment: {},
```

Comment:
Should we apply this change to other fields?

Reply:
No< we shouldn't. These should be empty objects in order to be able to edit correctly.